pkg/audit/newverifier_test.go

```diff
@@ -64,10 +64,10 @@ func TestVerifierHappyPath(t *testing.T) {
 		err = planet.StopPeer(planet.StorageNodes[1])
 		require.NoError(t, err)
 
-		// remove stopped nodes from overlay cache
-		err = planet.Satellites[0].Overlay.Service.Delete(ctx, planet.StorageNodes[0].ID())
+		// mark stopped nodes as offline in overlay cache
+		_, err = planet.Satellites[0].Overlay.Service.UpdateUptime(ctx, planet.StorageNodes[0].ID(), false)
 		require.NoError(t, err)
-		err = planet.Satellites[0].Overlay.Service.Delete(ctx, planet.StorageNodes[1].ID())
+		_, err = planet.Satellites[0].Overlay.Service.UpdateUptime(ctx, planet.StorageNodes[1].ID(), false)
```

Comment:
this also should be done in `pkg/storage/segments/repairer_test.go`

Reply:
Good point! I updated this test too.

Do you think we should enable TestSegmentStoreRepair back? It looks stable if I run it locally a few hundred times.

Reply:
Oh. I see you already enabled it with #1557.